sw/airborne/modules/gps/gps.c

```diff
@@ -80,6 +80,13 @@ static uint8_t current_gps_id = GpsId(PRIMARY_GPS);
 #endif
 
 uint8_t multi_gps_mode;
+int gps_disable_fix = 0;              ///< Disable fix
```

Comment:
Can you add some more detailed comments about this feature, as it is dangerous if wrongly used.
What do you think about adding a flag to disable the feature by default ?